api/index.js

```diff
@@ -35,6 +35,9 @@ async function _exitOnSignal(signal) {
   logger.info(`Received signal: ${signal}.`);
   logger.info('Stopping HAPI server...');
   await server.stop({ timeout: 30000 });
+  if (server.directMetrics) {
+    server.directMetrics.clearMetrics();
+  }
```

Comment:
```suggestion
  server.directMetrics?.clearMetrics();
```

ça fait pas le job de faire ça ?